src/index.ts

```diff
@@ -72,6 +72,7 @@ export function resolveInput(
 	resolvers: Resolver[],
 	resolveListeners: ResolveListeners,
 	dockerfile?: string,
+	additionalTemplateVars?: { [key: string]: string },
```

Comment:
`@types/lodash` has a `Dictionary` type:  `_.Dictionary<string>`

Reply:
I just added a global.d.ts so I didn't have to do the lodash import everywhere.